Sources/App/commands.swift

```diff
@@ -30,15 +31,74 @@ extension SlackCommand {
                             """
                         )
                 }
-        }
+        })
     }
 
     private static let branchOptionPrefix = "branch:"
 
+    static let crp = { (jiraService: JiraService, githubService: GitHubService) in
+        SlackCommand(
+            name: "crp",
+            help: """
+                Creates a ticket on the CRP board.
+
+                Example:
+                `/crp ios release/3.12-nhs111`
+                """,
+            token: Environment.get("SLACK_TOKEN")!,
+            run: { metadata, request in
+                let components = metadata.text.components(separatedBy: " ")
+                guard components.count >= 2 else {
+                    throw SlackService.Error.missingParameter(key: "repo, branch")
+                }
+
+                let repoName = components[0]
+                guard let repoID = RepoID.find(matching: repoName) else {
+                    let all = RepoID.allCases.map({ $0.rawValue }).joined(separator: "|")
+                    throw SlackService.Error.invalidParameter(key: "repo", value: repoName, expected: all)
+                }
+
+                let branchName = components[1]
+                let release = try GitHubService.Release(repository: repoID.repository, branchName: branchName)
+                return try githubService.changelog(for: release, on: request)
+                    .map { changelog in
+                        SlackCommand.makeCRPIssue(
```

Comment:
[nit] Tbh I was a bit confused that it is defined on `SlackCommand` until realised it is static. Seems less confusing without type.

```suggestion
                        makeCRPIssue(
```